users/replicaJunction/features/mouse_jiggle.c

```diff
@@ -0,0 +1,30 @@
+#include "mouse_jiggle.h"
+
+bool is_mouse_jiggle_active = false;
+
+void matrix_scan_mouse_jiggle(void) {
```

Comment:
If you need the mouse to just send reports constant to trick the system into staying awake ....  you could use pointing device support, and a custom "send" function to do this. 

```c
void pointing_device_send(void) {
    host_mouse_send(&mouseReport);
    mouseReport.x = 0;
    mouseReport.y = 0;
    mouseReport.v = 0;
    mouseReport.h = 0;
}
```
This will be considered activity, even though no movement occurs, and will keep the system awake.  In fact, I had to fix that issue because that was unwanted behavior.